### PR TITLE
Config file update

### PR DESCRIPTION
**Pull request by adamsthws**
# Enhance Clarity and Readability of Configuration Comments

## Overview

This pull request introduces a series of comment updates in the configuration files to improve clarity and readability. 

By enhancing the documentation directly within the configuration files, we can reduce the learning curve and help prevent configuration errors.

The changes should make it easier to setup log2ram, especially for new users or those not familiar with the more technical aspects of system configuration. 

The revisions provide concise, clear explanations of configuration settings. This includes specifying the purpose and implications of each setting, offering examples where applicable, and outlining the effects of different configurations on system behavior. 

Thankyou,
Adam

### CODE REVIEW

**Comment by azlux** [comment]
Sorry, I had to force push the branch (error on my side), can you rebase it ?

**Comment by adamsthws** [comment]
> Sorry, I had to force push the branch (error on my side), can you rebase it ?

Yes Sir, no problem!

File: log2ram.conf [hunk 1]
```diff
@@ -1,49 +1,66 @@
...
 # rsync is mandatory. Also make sure that you have configured SystemMaxUse in journald.conf, because the size of this
 # ram folder is will probably not be as large as journald will use by default
 JOURNALD_AWARE=true
 
-# **************** Zram backing conf  *************************************************
+# Set to 'true' to enable log rotation for journald logs before syncing. 
+# Note: 'rsync' must be used for this feature. 
+# Ensure 'SystemMaxUse' is configured in 'journald.conf' 
+#  - (To limit journald’s disk usage to be smaller than the log2ram RAM allocation).
+JOURNALD_AWARE=true
```
Comment (XhmikosR) [inline log2ram.conf:41]: Ins;t this block duplicated with line 35?

Comment (adamsthws) [reply log2ram.conf:41]: Yes, thankyou for catching this! It has now been fixed in: [this commit](https://github.com/azlux/log2ram/pull/227/commits/feccb1d3c0b5e2b8727db988bb1c24f16b4cc8af)

**Pull request merged**